packages/debug/src/browser/debug-prefix-configuration.ts

```diff
@@ -107,18 +107,42 @@ export class DebugPrefixConfiguration implements CommandContribution, CommandHan
         });
     }
 
-    getPicks(filter: string, token: CancellationToken): QuickPicks {
+    async getPicks(filter: string, token: CancellationToken): Promise<QuickPicks> {
         const items: QuickPickItem[] = [];
-        const configurations = this.debugConfigurationManager.all;
-        Array.from(configurations).forEach(config => {
+        const configurationManager = this.debugConfigurationManager;
```

Comment:
I don't believe there is a need to store this additional variable, we can use `this.debugConfigurationManager` directly.